apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/config/WebSecurityConfig.java

```diff
@@ -1,26 +1,76 @@
 package gov.cdc.nbs.questionbank.config;
 
+import java.io.IOException;	
+import javax.servlet.http.HttpServletRequest;	
+import javax.servlet.http.HttpServletResponse;	
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpStatus;	
+import org.springframework.security.authentication.InsufficientAuthenticationException;
 import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.core.AuthenticationException;	
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
 import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.authentication.preauth.RequestHeaderAuthenticationFilter;	
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import gov.cdc.nbs.questionbank.config.security.JWTFilter;
+import graphql.GraphQLError;	
+import graphql.GraphqlErrorBuilder;	
+import lombok.RequiredArgsConstructor;
+
 
 @Configuration
+@RequiredArgsConstructor
 @EnableWebSecurity
 @EnableGlobalMethodSecurity(prePostEnabled = true, securedEnabled = true)
 public class WebSecurityConfig {
+	 private final JWTFilter jwtFilter;	
+	    private final ObjectMapper mapper;	
+	    @Value("${spring.graphql.path: graphql}")	
+	    private String graphQLEndpoint;
 
     @Bean
     public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
         return http.authorizeRequests()
-                .anyRequest()
-                .permitAll()
+        		 .antMatchers("/graphql", "/encryption/*")	
```

Comment:
can remove the `"/encryption/*"` matcher. The encryption endpoints exist only on the modernization-api and are used to encrypting query parameters.

Reply:
will updated